src/xenia/base/filesystem_posix.cc

```diff
@@ -122,18 +121,16 @@ static int removeCallback(const char* fpath, const struct stat* sb,
 }
 
 bool DeleteFolder(const std::filesystem::path& path) {
-  return nftw(path.c_str(), removeCallback, 64, FTW_DEPTH | FTW_PHYS) == 0
-             ? true
-             : false;
+  return nftw(path.c_str(), removeCallback, 64, FTW_DEPTH | FTW_PHYS) == 0;
 }
 
-static uint64_t convertUnixtimeToWinFiletime(time_t unixtime) {
-  // Linux uses number of seconds since 1/1/1970, and Windows uses
+static uint64_t convertUnixtimeToWinFiletime(const timespec& unixtime) {
+  // Linux uses number of nanoseconds since 1/1/1970, and Windows uses
   // number of nanoseconds since 1/1/1601
-  // so we convert linux time to nanoseconds and then add the number of
-  // nanoseconds from 1601 to 1970
+  // so we add the number of nanoseconds from 1601 to 1970
   // see https://msdn.microsoft.com/en-us/library/ms724228
-  uint64_t filetime = filetime = (unixtime * 10000000) + 116444736000000000;
+  uint64_t filetime =
+      (unixtime.tv_sec * 10000000) + unixtime.tv_nsec + 116444736000000000;
```

Comment:
It's supposed to be in 100ns intervals.
`(unixtime.tv_sec * 10000000) + unixtime.tv_nsec / 100 + 116444736000000000`